back/.rubocop.yml

```diff
@@ -24,6 +24,7 @@ Layout/HashAlignment:
   EnforcedHashRocketStyle: [key, table]
 
 Metrics/BlockLength:
+  Max: 384
```

Comment:
Not blocking, but I wonder if rules like this are useful. I think we all know instinctively when something is too big/complex and the decision not to deal with it is usually accompanied by noting it as tech debt, and then increasing the number in a rule like this.

In effect the rule could just be slowing us down without adding value.